<!--
	This launch file should be run on the Pi in order to start our drive software.
 -->
<launch>

    <!-- params for hardware interfaces including IMU, GPS, and moteus -->
    <rosparam command="load" file="$(find mrover)/config/esw.yaml"/>

    <!-- Run the moteus program -->
    <node name="brushless" pkg="mrover" type="brushless.py" output="screen"></node>

    <!-- Run the auton led program -->
    <node name="auton_led" pkg="mrover" type="auton_led.py" output="screen"></node>

</launch>
